receiver/awscontainerinsightreceiver/internal/k8sapiserver/prometheus_helper_test.go

```diff
@@ -0,0 +1,128 @@
+// Copyright The OpenTelemetry Authors
```

Comment:
Probably want to add this copyright info to other new files as well

Reply:
Will do in https://github.com/amazon-contributing/opentelemetry-collector-contrib/pull/16